### PR TITLE
GRW-2016/Desktop layout - Cart Entry

### PR DESCRIPTION
**Pull request by KarnellSchultz**
## Describe your changes

Created a desktop layout of cart entry items. This component is seen on the /cart and /checkout pages.

Attempted to do this in several ways. Thought this was the least complicated, although I don't _love_ how the show/hide of things like the "title" (`<MobileTitle />` & `<DesktopTitle />`) are done with css. 

checkout-entries
cart-entries

## Justify why they are needed

Matching the designs in Figma.
## Figma
Screenshot 2023-01-05 at 15 07 31

## Checklist before requesting a review

- [x] I have performed a self-review of my code

[GRW-2016]: https://hedvig.atlassian.net/browse/GRW-2016?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/store/src/components/CartInventory/CartEntryItem.tsx [hunk 1]
```diff
@@ -1,58 +1,135 @@
...
+    display: 'grid',
+    gridTemplateColumns: '3fr 2fr 2fr',
+    gap: theme.space.sm,
+  },
+}))
+
+const PillowWrapper = styled.div({
+  display: 'flex',
+  alignItems: 'flex-start',
+  [mq.lg]: {
```
Comment (alebedev) [inline apps/store/src/components/CartInventory/CartEntryItem.tsx:112]: Doesn't this block duplicate styles above?

**Review by alebedev — COMMENTED** [review]
This probably needs a merge with PR separating EntryRemoveDialog (@source:lambda  START)

File: apps/store/src/components/CartInventory/CartEntryItem.tsx [hunk 2]
```diff
@@ -1,58 +1,124 @@
...
+  gap: theme.space.sm,
+  [mq.lg]: {
+    gridTemplateColumns: '3fr 2fr 2fr',
+  },
+}))
+
+const DesktopTitle = styled(Text)({
+  display: 'none',
+  [mq.lg]: {
+    display: 'block',
```
Comment (guilhermespopolin) [inline apps/store/src/components/CartInventory/CartEntryItem.tsx:110]:
```suggestion
    display: 'revert',
```
This is just a small suggestion. I've seen this "pattern" a lot recently. The advantage is that one doesn't need to worry about checking what would be the default display value of an element.

**Review by guilhermespopolin — COMMENTED** [review]
I think that's fine the way you've implemented show/hide titles with CSS. However I agree with Aleks that we probably want to reuse [`RemoveEntryDialog`](https://github.com/HedvigInsurance/racoon/pull/1260/files#diff-0b9d91d939c55dc07c7f4a6b04544f5c0a63eff33571b551273131208bcc9035) here.

File: apps/store/src/components/CartInventory/CartEntryItem.tsx [hunk 3]
```diff
@@ -1,58 +1,124 @@
...
-            </Dialog.Trigger>
-          </DetailsSheetDialog>
-          <RemoveEntryDialog cartId={cartId} {...cartEntry}>
+        <Space y={1}>
+          <div>
+            <MobileTitle size="md">{title}</MobileTitle>
+            <Text size="md" color="textSecondary">
+              {/* @TODO: display "automatically switches" if cancellation is requested" */}
+              {startDate
+                ? t('CART_ENTRY_DATE_LABEL', { date: formatter.fromNow(startDate), ns: 'cart' })
```
Comment (alebedev) [inline apps/store/src/components/CartInventory/CartEntryItem.tsx:45]: Nit: don't need explicit ns if it matches argument of `useTranslation()`

**Comment by KarnellSchultz** [comment]
@alebedev @guilhermespopolin  Thanks for the comments. Looks like I mixed up the merge conflicts when working on this PR and left out the stuff from Robins [PR](https://github.com/HedvigInsurance/racoon/pull/1260/files#).

Should be sorted now. I'm guessing some of this stuff is going to change as we get more information on breakpoints from designers.

**Review by alebedev — APPROVED** [review]
Looks good!

**Pull request merged**